integration_test/install_test.go

```diff
@@ -105,6 +111,95 @@ func TestKrewInstall_Manifest(t *testing.T) {
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
 
+func startServer() (*httptest.Server, error) {
+	server := httptest.NewUnstartedServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.RequestURI {
+		case "/konfig_localhost.yaml":
```

Comment:
why don't you just use http.FileServer ?

Reply:
I thought having a custom handler function will be helpful if we need to serve additional files in the future from different directories.

Reply:
But we currently don’t, and this implementations is 20x longer. :)

Reply:
I will fix it

Reply:
Done